css-three-column-layout/styles.css

```diff
@@ -0,0 +1,77 @@
+body {
+  background-color: rgb(25, 150, 218);
+  font-family: Lato, sans-serif;
+  height: 100vh;
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
I wouldn't apply `display: flex` on your `body`. I would save this for `row` or row type elements.  For the most part, you want to avoid applying styling to the `body`. We want to style elements that are visible on the page, the body is not visible on the page.

Reply:
Updated